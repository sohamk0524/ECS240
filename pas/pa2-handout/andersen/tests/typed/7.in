c Test case with mix of all statements
p 105 160
t 1 20
t 2 20
t 3 20
t 4 20
t 5 20
t 6 19
t 7 19
t 8 19
t 9 19
t 10 19
t 11 18
t 12 18
t 13 18
t 14 18
t 15 18
t 16 17
t 17 17
t 18 17
t 19 17
t 20 17
t 21 16
t 22 16
t 23 16
t 24 16
t 25 16
t 26 15
t 27 15
t 28 15
t 29 15
t 30 15
t 31 14
t 32 14
t 33 14
t 34 14
t 35 14
t 36 13
t 37 13
t 38 13
t 39 13
t 40 13
t 41 12
t 42 12
t 43 12
t 44 12
t 45 12
t 46 11
t 47 11
t 48 11
t 49 11
t 50 11
t 51 10
t 52 10
t 53 10
t 54 10
t 55 10
t 56 9
t 57 9
t 58 9
t 59 9
t 60 9
t 61 8
t 62 8
t 63 8
t 64 8
t 65 8
t 66 7
t 67 7
t 68 7
t 69 7
t 70 7
t 71 6
t 72 6
t 73 6
t 74 6
t 75 6
t 76 5
t 77 5
t 78 5
t 79 5
t 80 5
t 81 4
t 82 4
t 83 4
t 84 4
t 85 4
t 86 3
t 87 3
t 88 3
t 89 3
t 90 3
t 91 2
t 92 2
t 93 2
t 94 2
t 95 2
t 96 1
t 97 1
t 98 1
t 99 1
t 100 1
t 101 0
t 102 0
t 103 0
t 104 0
t 105 0
s 0 1 -1 7
s 0 5 -1 6
s 0 2 -1 8
s 0 2 0 2
s 0 1 0 1
s 0 2 0 1
s 0 5 0 2
s 0 1 0 2
s 0 6 -1 12
s 0 10 -1 11
s 0 9 -1 11
s 0 9 1 2
s 0 10 0 10
s 1 1 1 2
s 1 5 0 6
s 1 2 1 5
s 0 11 -1 19
s 0 15 -1 18
s 0 14 -1 16
s 2 5 1 6
s 1 10 0 14
s 2 1 1 6
s 2 1 0 14
s 0 15 1 6
s 0 16 -1 23
s 0 20 -1 24
s 0 19 -1 21
s 1 11 0 19
s 0 20 0 16
s 3 2 3 5
s 1 15 3 5
s 0 16 3 2
s 0 21 -1 30
s 0 25 -1 29
s 0 23 -1 30
s 3 9 1 20
s 0 21 4 1
s 2 11 1 19
s 3 10 4 2
s 2 15 2 14
s 0 26 -1 32
s 0 30 -1 33
s 0 27 -1 32
s 4 6 4 6
s 0 27 5 1
s 1 23 0 27
s 2 16 1 23
s 5 2 1 23
s 0 31 -1 37
s 0 35 -1 38
s 0 34 -1 38
s 6 5 0 34
s 4 11 3 16
s 5 6 6 5
s 4 15 4 14
s 2 25 5 6
s 0 36 -1 44
s 0 40 -1 42
s 0 37 -1 42
s 4 19 0 36
s 1 31 2 30
s 1 35 0 37
s 3 21 6 9
s 4 19 0 37
s 0 41 -1 46
s 0 45 -1 49
s 0 43 -1 48
s 6 14 0 41
s 5 20 3 30
s 4 25 1 37
s 8 1 0 45
s 4 23 6 15
s 0 46 -1 52
s 0 50 -1 51
s 0 47 -1 51
s 0 46 0 46
s 4 26 7 11
s 7 14 3 35
s 5 25 2 40
s 9 5 6 20
s 0 51 -1 56
s 0 55 -1 59
s 0 53 -1 56
s 4 35 1 47
s 6 25 6 23
s 2 43 2 41
s 9 9 10 1
s 6 23 0 51
s 0 56 -1 63
s 0 60 -1 61
s 0 59 -1 65
s 7 25 7 23
s 6 30 11 5
s 7 23 1 55
s 4 36 2 47
s 11 5 9 11
s 0 61 -1 66
s 0 65 -1 69
s 0 63 -1 66
s 3 47 0 65
s 1 59 5 37
s 6 34 11 6
s 11 6 8 25
s 12 1 12 5
s 0 66 -1 73
s 0 70 -1 75
s 0 68 -1 75
s 6 36 8 30
s 3 53 0 70
s 1 63 3 55
s 0 68 3 51
s 1 65 8 30
s 0 71 -1 79
s 0 75 -1 80
s 0 72 -1 76
s 10 23 2 61
s 0 75 7 36
s 9 30 7 36
s 7 37 3 56
s 4 55 9 26
s 0 76 -1 85
s 0 80 -1 83
s 0 79 -1 84
s 4 59 10 26
s 5 53 7 43
s 5 51 1 72
s 13 14 6 47
s 6 47 7 41
s 0 81 -1 88
s 0 85 -1 86
s 0 84 -1 86
s 0 85 10 34
s 3 70 16 1
s 7 50 13 19
s 10 34 8 41
s 7 46 12 25
s 0 86 -1 94
s 0 90 -1 95
s 0 88 -1 95
s 8 50 4 68
s 3 72 4 68
s 1 81 12 26
s 3 72 16 6
s 2 76 2 76
s 0 91 -1 98
s 0 95 -1 99
s 0 92 -1 100
s 15 20 11 37
s 16 11 18 5
s 4 72 6 61
s 7 59 1 86
s 7 60 14 21
s 0 96 -1 105
s 0 100 -1 103
s 0 98 -1 103
s 10 47 6 66
s 18 10 15 25
s 15 21 9 51
s 11 43 0 100
s 7 63 15 21
