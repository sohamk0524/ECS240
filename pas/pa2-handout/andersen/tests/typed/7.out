pt 100 103
pt 10 11
pt 11 16
pt 11 19
pt 12 16
pt 14 16
pt 15 16
pt 15 18
pt 16 21
pt 16 23
pt 1 7
pt 1 8
pt 18 21
pt 18 23
pt 19 21
pt 20 21
pt 20 23
pt 20 24
pt 21 30
pt 23 30
pt 25 29
pt 26 32
pt 2 7
pt 27 32
pt 27 33
pt 2 8
pt 30 32
pt 30 33
pt 31 37
pt 32 38
pt 33 38
pt 34 38
pt 35 38
pt 36 44
pt 37 42
pt 37 44
pt 38 42
pt 38 44
pt 40 42
pt 41 46
pt 42 46
pt 42 49
pt 43 48
pt 44 46
pt 44 49
pt 45 49
pt 46 52
pt 47 51
pt 50 51
pt 51 56
pt 52 56
pt 53 56
pt 55 59
pt 5 6
pt 56 63
pt 56 65
pt 5 7
pt 5 8
pt 59 65
pt 60 61
pt 6 12
pt 61 66
pt 63 66
pt 63 69
pt 65 66
pt 65 69
pt 66 73
pt 66 75
pt 68 73
pt 68 75
pt 69 73
pt 69 75
pt 70 75
pt 7 12
pt 71 79
pt 72 76
pt 73 80
pt 75 80
pt 76 85
pt 79 84
pt 80 83
pt 80 85
pt 8 12
pt 81 88
pt 84 86
pt 85 86
pt 86 94
pt 88 94
pt 88 95
pt 90 95
pt 9 11
pt 9 12
pt 91 98
pt 92 100
pt 95 99
pt 96 105
pt 98 103
